<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MyWidgetNEW" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>80</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="MyWidgetNEW" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>80</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>60</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>от A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>60</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>до B</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>141</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Способ ввода</string>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>счётчик</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>ползунок</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>установщик</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>321</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 211, 211);
border-radius: 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>211</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Запись результата в файл</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>121</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 5px;
background-color: rgb(255, 255, 255);
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Размер массива</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>231</width>
      <height>361</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(157, 157, 157);
border-color: rgb(255, 88, 233);
border-radius: 10px;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>131</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="title">
      <string>Выбор задания</string>
     </property>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>121</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>b) Второе задание</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>121</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>a) Первое задание</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Значение С</string>
     </property>
    </widget>
    <zorder>groupBox_2</zorder>
    <zorder>label_4</zorder>
    <zorder>lineEdit_2</zorder>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>370</y>
      <width>321</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 10px;
color: rgb(0, 0, 0);
background-color: rgb(114, 255, 239);</string>
    </property>
    <property name="text">
     <string>Запуск алгоритма</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>200</y>
      <width>321</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 15px;
background-color: rgb(255, 255, 255);
border-color: rgb(255, 0, 4);</string>
    </property>
   </widget>
   <zorder>frame_2</zorder>
   <zorder>frame</zorder>
   <zorder>widget</zorder>
   <zorder>widget_2</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>checkBox</zorder>
   <zorder>lineEdit</zorder>
   <zorder>label_3</zorder>
   <zorder>pushButton</zorder>
   <zorder>textEdit</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyWidgetNEW</class>
   <extends>QWidget</extends>
   <header>mywidgetnew.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
